lib/Whim.pm

```diff
@@ -18,72 +13,20 @@ sub startup {
     $self->helper(
         whim => sub {
             state $whim = Whim::Core->new(
-                { data_directory => "$FindBin::Bin/../data" } );
+                {   data_directory => "$FindBin::Bin/../data",
```

Comment:
With `$FindBin::Bin/..` getting reused, maybe set it as a package variable to reduce fat-finger errors and clarify intent.

``` perl
our $APP_DIR = "$FindBin::Bin/..";
```

Then around here something like

``` perl
{ data_directory => "$APP_DIR/data", author_photo_directory => "$APP_DIR/public/author_photos" }
```

Reply:
I'll do you one better and just drop FindBin from this context, swapping in the mojo object's `home` method.